react/features/base/config/configWhitelist.js

```diff
@@ -158,6 +158,7 @@ export default [
     'fileRecordingsEnabled',
     'filmstrip',
     'firefox_fake_device',
+    'flags',
```

Comment:
is that enough? does it also need `flags.full-flag-name`? I think I had to add the full name when testing locally, but may be missing something

Reply:
I am pretty sure that should be enough.